pkg/cmd/controller/security.go

```diff
@@ -29,13 +32,20 @@ func RunNamespaceSecurityAllocationController(ctx context.Context, controllerCtx
 		return true, err
 	}
 
+	metrics, err := sccmetrics.NewNamespaceSCCAllocMetrics()
```

Comment:
If `NamespaceSCCAllocationController` is the only consumer of these metrics, I'm inclined to register them directly inside that controller, and use `MustRegister` which will panic if this doesn't work. That's what we do in k8s controllers, see https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/cronjob/cronjob_controllerv2.go#L122

Reply:
I have moved them inside the controller, but kept the NamespaceSCCAllocationMetrics interface for easier testing